packages/lit-dev-content/site/docs/ssr/server-api.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Lit SSR server API
+eleventyNavigation:
+  key: Server API
+  parent: Server-side rendering
+  order: 2
+---
+
+Lit SSR provides a couple different ways of rendering custom elements server-side: rendering in the [global scope](#global-scope) or via [VM modules](#vm-module).
```

Comment:
I'm unsure about focusing purely on how the DOM shim is loaded as being the sole differentiator between the two ways since there may be other reasons to prefer isolation.

My original take was that we have two ways of rendering, global and isolated in VM modules where how the DOM shim is loaded happens to be one difference, albeit a significant one, between the two.

All of our existing SSR doc and code refer to adding the DOM APIs as "DOM Shim". @justinfagnani suggested a section called "DOM Compatibility (or DOM Emulation)". In this draft I've been using them interchangeably though maybe we should just stick to DOM Shim consistently instead?

With regards to VM modules, perhaps we can say something along the lines of
`The 'renderModule()` function utilizes Node's [`vm.Module` class](https://nodejs.org/api/vm.html#class-vmmodule) which enables running code within V8 Virtual Machine contexts.

Reply:
OK. Perhaps I missed something in reading the section, I just didn't see any other differences mentioned. You could soften my earlier suggestion a bit like this:

```suggestion
Lit SSR provides two ways of rendering custom elements server-side: rendering in the [global scope](#global-scope) or using[VM modules](#vm-module). The two methods differ primarily in how the DOM shim is loaded.
```

I don't think the section title is a problem, but if you use both DOM emulation and DOM shim in this section, I'd just try to make it really clear that you were talking about the same thing.